core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20211115_v1_workspace_migration_history.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="ehigham" id="create_V1_WORKSPACE_MIGRATION_HISTORY">
+        <createTable tableName="V1_WORKSPACE_MIGRATION_HISTORY"
```

Comment:
How does Rawls keep track of what step of the migration a workspace is on?

Reply:
We add datetime columns to this table for each step. I'm not sure what these are yet so I haven't included them